examples/wheel/BUILD.bazel

```diff
@@ -321,6 +335,7 @@ py_test(
         ":custom_package_root_multi_prefix_reverse_order",
         ":customized",
         ":filename_escaping",
+        ":minimal_data_files",
```

Comment:
I would expect an addition of a test under `wheel_test.py` which checks the directory structure, similarly to how the existing tests do. Right now we are just building the test wheel but we don't have any test verifying that the data files end up in the right locations.

Reply:
Oh, I hadn't followed up on what that py-test rule was doing.  Thanks, and added.